CommunityTracker/CommunityTracker.Service/Commands/AddCommunity.cs

```diff
@@ -73,5 +70,26 @@ private async Task<CommunityResponseDTO> MapAddCommunityResponse(CommunityDTO co
 
             return result;
         }
+
+        private async Task<CommunityResponseDTO> ResponseValidation(CommunityDTO communityDTO)
+        {
+            var community = new CommunityResponseDTO();
+
+            var communities = await _communityRepositoryQuery.GetAllCommunities();
+            var managers = await _communityRepositoryQuery.GetAllManagers();
+
+            //checks if community is existing
+            bool communityExists = communities.Any(x => x.CommunityName.ToLower() == communityDTO.CommunityName.ToLower());
+
+            //checks if managerId is existing
+            bool managerIdExists = managers.Any(x => x.CommunityAdminAndManagerId == communityDTO.CommunityMgrid);
+
+            if (communityExists || !managerIdExists)
+            {
+                return null;
+            }
+
+            return community;
+        }
```

Comment:
```suggestion
        }
```

Please move this method to a separate public class method